assets/app/features/upgrade/controllers/upgrade-database-configuration.controller.js

```diff
@@ -41,7 +46,13 @@
         vm.createServer = createServer;
         vm.connectServer = connectServer;
 
+        activate();
+
         // functions
+        function activate() {
+            upgradeStatusFactory.syncStatusFlags(UPGRADE_STEPS.database, vm,
+                undefined, upgradeStepsFactory.setCurrentStepCompleted);
```

Comment:
shouldnt we pass a function here to `onRunning` so if its running we can disable buttons and so on?